vm/vm_test.go

```diff
@@ -362,6 +362,56 @@ func splitFullValueStateTransition(t *testing.T, validRootLocator,
 	}
 }
 
+func splitCollectibleStateTransition(t *testing.T,
```

Comment:
nit: the outer `t` isn't needed. But I see that it's there in the other test generator functions too, so non blocking.